<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ToolWidget</class>
 <widget class="QWidget" name="ToolWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>507</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>250</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>250</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Tools</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="createTab">
      <attribute name="title">
       <string>Create</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QToolBox" name="toolBox">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="basicPage">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>180</width>
            <height>360</height>
           </rect>
          </property>
          <attribute name="label">
           <string>Primitive</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <layout class="QGridLayout" name="gridLayout" rowstretch="0,8">
             <property name="horizontalSpacing">
              <number>4</number>
             </property>
             <property name="verticalSpacing">
              <number>15</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <item row="0" column="0">
              <widget class="QPushButton" name="cubeButton">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="maximumSize">
                <size>
                 <width>32</width>
                 <height>32</height>
                </size>
               </property>
               <property name="toolTip">
                <string>Cube</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="resource.qrc">
                 <normaloff>:/image/images/cube.png</normaloff>:/image/images/cube.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>32</width>
                 <height>32</height>
                </size>
               </property>
               <property name="checkable">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QPushButton" name="sphereButton">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="maximumSize">
                <size>
                 <width>32</width>
                 <height>32</height>
                </size>
               </property>
               <property name="toolTip">
                <string>Sphere</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="resource.qrc">
                 <normaloff>:/image/images/sphere.png</normaloff>:/image/images/sphere.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>32</width>
                 <height>32</height>
                </size>
               </property>
               <property name="checkable">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <widget class="QPushButton" name="cylinderButton">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="maximumSize">
                <size>
                 <width>32</width>
                 <height>32</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="resource.qrc">
                 <normaloff>:/image/images/cylinder.png</normaloff>:/image/images/cylinder.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>32</width>
                 <height>32</height>
                </size>
               </property>
               <property name="checkable">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item row="0" column="3">
              <widget class="QPushButton" name="coneButton">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="maximumSize">
                <size>
                 <width>32</width>
                 <height>32</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="resource.qrc">
                 <normaloff>:/image/images/cone.png</normaloff>:/image/images/cone.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>32</width>
                 <height>32</height>
                </size>
               </property>
               <property name="checkable">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QPushButton" name="prismButton">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="maximumSize">
                <size>
                 <width>32</width>
                 <height>32</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="resource.qrc">
                 <normaloff>:/image/images/prism.png</normaloff>:/image/images/prism.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>32</width>
                 <height>32</height>
                </size>
               </property>
               <property name="checkable">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QPushButton" name="pyramidButton">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="maximumSize">
                <size>
                 <width>32</width>
                 <height>32</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="resource.qrc">
                 <normaloff>:/image/images/pyramid.png</normaloff>:/image/images/pyramid.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>32</width>
                 <height>32</height>
                </size>
               </property>
               <property name="checkable">
                <bool>false</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="boolPage">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>98</width>
            <height>28</height>
           </rect>
          </property>
          <attribute name="label">
           <string>Bool</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <layout class="QGridLayout" name="gridLayout_2"/>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="modifyTab">
      <attribute name="title">
       <string>Modify</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Solid: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="modelBox"/>
       </item>
       <item>
        <widget class="QWidget" name="paraWidget" native="true">
         <layout class="QGridLayout" name="gridLayout_3">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="widthLabel">
            <property name="text">
             <string>Width:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QDoubleSpinBox" name="widthSpin">
            <property name="minimum">
             <double>0.100000000000000</double>
            </property>
            <property name="maximum">
             <double>100.000000000000000</double>
            </property>
            <property name="value">
             <double>1.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="heightLabel">
            <property name="text">
             <string>Height: </string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QDoubleSpinBox" name="heightSpin">
            <property name="minimum">
             <double>0.100000000000000</double>
            </property>
            <property name="maximum">
             <double>100.000000000000000</double>
            </property>
            <property name="value">
             <double>1.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="depthLabel">
            <property name="text">
             <string>Depth: </string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QDoubleSpinBox" name="depthSpin">
            <property name="minimum">
             <double>0.100000000000000</double>
            </property>
            <property name="maximum">
             <double>100.000000000000000</double>
            </property>
            <property name="value">
             <double>1.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="8" column="1">
           <widget class="QPushButton" name="okButton">
            <property name="text">
             <string>OK</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="radiusLabel">
            <property name="text">
             <string>Radius:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QDoubleSpinBox" name="radiusSpin">
            <property name="minimum">
             <double>0.100000000000000</double>
            </property>
            <property name="maximum">
             <double>100.000000000000000</double>
            </property>
            <property name="value">
             <double>0.500000000000000</double>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="lengthLabel">
            <property name="text">
             <string>Length: </string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QDoubleSpinBox" name="lengthSpin">
            <property name="minimum">
             <double>0.100000000000000</double>
            </property>
            <property name="maximum">
             <double>100.000000000000000</double>
            </property>
            <property name="value">
             <double>1.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="sideLabel">
            <property name="text">
             <string>Side length:</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QDoubleSpinBox" name="sideSpin">
            <property name="minimum">
             <double>0.100000000000000</double>
            </property>
            <property name="maximum">
             <double>100.000000000000000</double>
            </property>
            <property name="value">
             <double>1.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="slicesLabel">
            <property name="text">
             <string>Slices: </string>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QSpinBox" name="slicesSpin">
            <property name="minimum">
             <number>3</number>
            </property>
            <property name="maximum">
             <number>256</number>
            </property>
            <property name="value">
             <number>16</number>
            </property>
           </widget>
          </item>
          <item row="7" column="0">
           <widget class="QLabel" name="stacksLabel">
            <property name="text">
             <string>Stacks: </string>
            </property>
           </widget>
          </item>
          <item row="7" column="1">
           <widget class="QSpinBox" name="stacksSpin">
            <property name="minimum">
             <number>2</number>
            </property>
            <property name="maximum">
             <number>256</number>
            </property>
            <property name="value">
             <number>8</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
